imap_processing/mag/l0/mag_l0_data.py

```diff
@@ -15,13 +15,12 @@ class Mode(IntEnum):
 
     Attributes
     ----------
-    BURST : int
-        APID for Burst mode data
-    NORMAL : int
-        ApID for Normal mode data
+    IntEnum : IntEnum
```

Comment:
I think the previous attributes should stay here unless it breaks the output

Reply:
I am skipping this because of those inherit methods warning. So I can undo this.